worlds/sc2wol/Locations.py

```diff
@@ -15,485 +15,710 @@ class LocationData(NamedTuple):
     region: str
     name: str
     code: Optional[int]
+    category: str
```

Comment:
I'd use Enum, like we use for mission pools. Less chance to leave a typo

Reply:
It will be done in the next commit. (TIL Enum, might not be using them perfectly)